test/k8s_logging_tests/test_config_logging.py

```diff
@@ -108,34 +108,6 @@ def test_annotation_excluding(setup, container_name, expected):
                 len(events))
     assert len(events) == expected
 
-@pytest.mark.skipif(True, reason="Jira: ADDON-36296")
```

Comment:
this ticket is closed. do we know the tests still need to be skipped?

Reply:
it was closed for inactivity.